src/applications/facility-locator/components/SearchResultsHeader.jsx

```diff
@@ -86,7 +86,6 @@ SearchResultsHeader.propTypes = {
   facilityType: PropTypes.string,
   serviceType: PropTypes.string,
   context: PropTypes.string,
-  specialtyMap: PropTypes.object,
```

Comment:
Can we add a `PropTypes` declaration for `specialties`?

Reply:
ok